src/models/Asset.js

```diff
@@ -151,3 +147,5 @@ export type AssetDataNavigationParam = {
   decimals: number,
   chain: Chain,
 }
+
+export type SupportedAssetsPerChain = ChainRecord<Asset[]>
```

Comment:
Makes sense, I would rename it to `AssetsPerChain` then.

Reply:
Yeah sounds ok. For consistency I would suggest a rename from `AssetsBySymbol` to `AssetBySymbol` though, as it can get confusing whether the object values are `Asset` or `Asset[]`. Wdyt?